resources/teamcity/android/keyman-android-release.sh

```diff
@@ -0,0 +1,128 @@
+
+builder_describe \
+  "Build release of Keyman for Android" \
+  "all            run all actions" \
+  "build          configure and build Keyman for Android" \
+  "publish        publish release" \
+  "--fv                               additionally build FV Android" \
+  "--rsync-path=RSYNC_PATH            rsync path on remote server" \
+  "--rsync-user=RSYNC_USER            rsync user on remote server" \
+  "--rsync-host=RSYNC_HOST            rsync host on remote server" \
+  "--rsync-root=RSYNC_ROOT            rsync root on remote server" \
+  "--help.keyman.com=HELP_KEYMAN_COM  path to help.keyman.com repository"
+
+builder_parse "$@"
+
+cd "${KEYMAN_ROOT}/android"
+
+function _publish_to_downloads_keyman_com() {
+  # Publish to downloads.keyman.com
+  builder_echo start "publish to downloads.keyman.com" "Publishing release to downloads.keyman.com"
+
+  local UPLOAD_PATH KEYMAN_ENGINE_ANDROID_ZIP KEYMAN_APK FIRSTVOICES_APK
+  local COMPRESS_CMD
+
+  # shellcheck disable=SC2154
+  UPLOAD_PATH="upload/${KEYMAN_VERSION}"
+  KEYMAN_ENGINE_ANDROID_ZIP="keyman-engine-android-${KEYMAN_VERSION}.zip"
+  KEYMAN_APK="keyman-${KEYMAN_VERSION}.apk"
+  FIRSTVOICES_APK="firstvoices-${KEYMAN_VERSION}.apk"
+
+  # shellcheck disable=SC2154
+  COMPRESS_CMD="${SEVENZ_HOME}/7z"
+
+  rm -rf "${UPLOAD_PATH}"
+  mkdir -p "${UPLOAD_PATH}"
+
+  (
+    cd KMAPro/kMAPro/libs
+    "${COMPRESS_CMD}" a -bd -bb0 "../../../${UPLOAD_PATH}/${KEYMAN_ENGINE_ANDROID_ZIP}" keyman-engine.aar ../../../Samples '-xr!build.sh'
+  )
+
+  cp "KMAPro/kMAPro/build/outputs/apk/release/${KEYMAN_APK}" "${UPLOAD_PATH}"
+
+  write_download_info "${UPLOAD_PATH}" "${KEYMAN_ENGINE_ANDROID_ZIP}" "Keyman Engine for Android" zip android
+  write_download_info "${UPLOAD_PATH}" "${KEYMAN_APK}" "Keyman for Android" apk android
+
+  if [[ -d "../oem/firstvoices/android/app/build/outputs/apk/release" ]]; then
+    cp "../oem/firstvoices/android/app/build/outputs/apk/release/${FIRSTVOICES_APK}" "${UPLOAD_PATH}"
+    write_download_info "${UPLOAD_PATH}" "${FIRSTVOICES_APK}" "FirstVoices Keyboards" apk android
+  fi
+
+  (
+    cd upload
+    # shellcheck disable=SC2154
+    tc_rsync_upload "${KEYMAN_VERSION}" "android/${KEYMAN_TIER}"
+  )
+
+  builder_echo end "publish to downloads.keyman.com" success "Finished publishing release to downloads.keyman.com"
+}
+
+function _publish_to_playstore() {
+  builder_echo start "publish to Google Play Store" "Publishing release to Google Play Store"
+
+  "${KEYMAN_ROOT}/android/build.sh" "publish:${PUBTARGETS}" --ci
+
+  builder_echo end "publish to Google Play Store" success "Finished publishing release to Google Play Store"
+}
+
+function do_build() {
+  android_build_action
+}
+
+function do_publish() {
+  if ! is_windows; then
+    # requires Powershell, so currently only supported on Windows
+    builder_echo error "This script is intended to be run on Windows only."
+    return 1
+  fi
```

Comment:
No longer uses powershell, right? But we need to test on non-Windows once we have finished moving the scripts over.

```suggestion
  if ! is_windows; then
    # currently only tested on Windows, TODO: test cross-platform
    builder_echo error "This script is intended to be run on Windows only."
    return 1
  fi
```

Reply:
done